module/multimedia/src/Domain/Query/MultimediaQueryInterface.php

```diff
@@ -14,7 +14,7 @@
 
 interface MultimediaQueryInterface
 {
-    public function fileExists(Hash $hash): bool;
+    public function fileExists(string $name): bool;
```

Comment:
The method name is incorrect. It does not check the existence of the file. I'd suggest deprecating both methods `fileExists` and `findIdByHash` and use findIdByFilename instead of fileExits